### DISCUSSION TITLE
Question about using IBEX for immunofluorescence of adherent murine cell line on 96 well plate.

### DISCUSSION THREAD

**Discussion by Sai101396** [IBEXImagingCommunity/ibex_imaging_knowledge_base#244] [Q&A]
Hello, I am a second-year PhD Student at UTMB (Galveston, Texas). For most of the IBEX protocols, I see they are based on tissues. Is there a protocol for just cells? I want to try using it for immunocytochemistry. The cells (in 96 wells) have already been plated, fixed, used with primary antibodies and screened with Cell Mask Blue, 488 and 568. I want to try to use the same IBEX protocol for the cells to de-stain and re-stain. Any suggestions or feedback is greatly appreciated.

**Comment by radtkea** [comment]
Welcome to the community! We are so glad you are here and are happy to support your PhD work. 

Yes, published work on IBEX has focused on tissues but there should be no reason why the method would not work on plated cells. This is a very interesting application, and I am excited for you to get it to work!

We have never used Cell Mask Blue, and therefore I am not sure if it inactivates with LiBH4. Ideally, it would be great if the signal remained, and you could use it as a fiducial for each round. I suspect that Cell Mask Red might not dye inactivate and could serve as a fiducial. 

AF488 dye inactivates but AF568 does not. You can see the full list of fluorescent probes tested [here](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/fluorescent_probes.html). There are several alternatives to AF568 that you can try such as AF555, iF555, PE, and eF570.

In conclusion:
1. You will need directly conjugated antibodies in AF488 or AF555/PE/iF555/eF570.
2. You will need to test if Cell Mask Blue dye inactivates. If yes, then you will need a different fiducial (DAPI/Hoechst or Cell Mask Red?). If no, then you can use it as a fiducial.
3. Hopefully, fixed cells are adherent to the coverslips over multiple rounds of imaging. If not, then you might need to pre-coat the coverslips with an adhesive. TBD.

Happy to chat some more, if you would like.

Best wishes,
Andrea

**Reply by Sai101396** [reply]
Hello Andrea

Thank you so much for your response and help! So the cells are already fixed into the 96 well plate wells (no cover slips, the optical plate is good for immunocytochemistry for imaging). My first round of antibodies included Paxillin and Synaptopodin. For the fluorescence, I used Cell Mask Blue, Goat anti Rabbit 488 (IgG) for the Paxillin and Goat anti Mouse 568 IgG for Synaptopodin. So in the 96 well plate, the first time I tried it with my plate was by washing with 1X PBS, 3 times (100 uL/well). Then I made the 1 mg/mL LiBH4 (0.005g of Lithium Borohydride to 5ml Millipore water - this is how I was told in the lab to make it). I then added 80 uL of the 1 mg/mL LiBH4 per well. I then did the photobleaching with the AF488 and red channel (using the mircoscope), for each for about 5 minutes. I tried blocking and restaining with another set of antibodies, but the fluorescence is still there. 

Even if the Cell Mask Blue does not inactivate, I think that should be fine, meaning the next round, I could just re-block and use another set of Abs.

So, I am in the midst of trying to plan and optimize this to see if I can get this to work for my immunofluorescence. Any suggestions/advice/feedback is greatly appreaciated! Also what is your opinion on using 1 mg/ml Sodium
borohydride (NaBH4) in PBS? Would this reagent work? 

Thank you, 
Sai

**Reply by radtkea** [reply]
Hi Sai,

Of course. Happy to help!

**Workflow:**
1. Cell Mask Blue, Paxillin/Goat anti-rabbit AF488, Synaptopodin/Goat anti-mouse IgG AF568
2. Dye inactivation 1 mg/ml for 5 minutes with photobleaching (microscope)
3. Blocking/restaining with another round of antibodies: Signal still present.

**Questions, observations, and suggestions:**
1. Observation: Your dye inactivation protocol is different from our most common method to dye inactivate, [1 mg/ml for 15 minutes](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/fluorescent_probes.html). Question: Did you observe complete signal loss when you had your specimen on the microscope? Suggestion: Redo the experiment and note under what conditions total signal is lost, e.g., 1 mg/ml for 15 minutes or 1 mg/ml for 5 minutes + photobleaching.
2. Observation: You are using a primary/secondary to detect the reagents in the first imaging round. Antibodies are not "stripped" from the tissue with the IBEX dye inactivation method. Therefore, when you "reprobe" with another round of antibodies, you might be detecting the first set of antibodies if you are using mouse and rabbit unconjugated primary antibodies and anti-mouse and anti-rabbit secondary antibodies. Question: What antibodies are you applying in the second round? Are they direct conjugates?

Sodium borohydride: We never tried it but has historically been used to reduce autofluorescence. Other colleagues use H2O2, cyCIF and Cell DIVE method, for imaging cells.

Here are a few protocols for multiplexed imaging of cells. Helpful resources for validated antibodies:
1. cyCIF: https://pmc.ncbi.nlm.nih.gov/articles/PMC5233430/
2. 4i: https://www.science.org/doi/abs/10.1126/science.aar7042

Best of luck! Please keep me posted on your progress.
Andrea

**Reply by Sai101396** [reply]
Hello Andrea, 
Happy Holidays! 

I ended up trying another plate that I had, where I previously used alpha tubulin (mouse) (used goat anti mouse 488) and Phalloidin 568 conjugated for Actin and cell mask blue. I added 100 uL of LiBH4 per well and left it under the LED lamp that we have for 45 minutes. I checked under the microscope that the 488 looked inactive and the red (568) did not inactivate, Cell mask blue still did not inactivate (which is fine). Afterward, I re-blocked with Goat serum, added ITGB5 (rabbit) primary Ab, incubate overnight at 4C and then washed and added Goat anti-rabbit 488 as secondary (for the ITGB5 primary Ab). When I went to image for the second round of antibody, I saw that i still saw a signal of alpha-tubulin and was not able to visualize ITGB5 (which would be around the nucleus). 

Next, I tried de-staining again with LiBH4 and incubated the plate under the LED lamp (in the cold room) overnight. I checked under the microscope the next day, and saw that were was signal loss of both 488 and 568. I then went ahead an re-blocked with Goat serum and added ITGB5 (rabbit) and re-stained with Goat-anti rabbit 488 again. when I went to image, I was able to see ITGB5, but still saw some signal of 488 (or I guess auto-fluoresence of alpha-tubulin?). 

I also have notice that there has been some cell loss, meaning that the cell density has decreased compared the previous fixed/stained imaged. I also observed a difference in cell shape in the healthy wells as well, as in they seem have structure loss comapred to the previous images.

Thank you!
Sai

**Reply by radtkea** [reply]
Hi Sai!

Happy Holidays to you too! I hope you are having a wonderful and joy-filled holiday season! 

Thank you very much for all these details! It is clear you are working hard:) You are also reproducing our results: AF568 is hard to inactivate.

I would be happy to chat with you via Teams/Zoom. I think this would be easier than multiple, asynchronous discussion threads. My email is andrea.radtke 'at' leica-microsystems.com. I am available December 30th or 31st, January 3rd, or after January 6th.  Late afternoon (US ET) is best for me. No pressure. 

Have a wonderful 2025,
Andrea
